build-tools/xabootstrap/xabootstrap/ConfigAndData/BuildAndroidPlatforms.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections.Generic;
+
+namespace Xamarin.Android.Bootstrap
+{
+	class BuildAndroidPlatforms
+	{
+		public const string AndroidNdkVersion = "19c";
+
+		public static readonly List<AndroidPlatform> AllPlatforms = new List<AndroidPlatform> {
+			// API, ID,   Framework, Stable
+			{ 0,    "0",  null,      false },
```

Comment:
Likewise here: I'd never considered the ability to add a `List<AndroidPlatform>.Add(...)` extension method before.  While "interesting", I'm not sure it's clearer than the alternative of `new AndroidPlatform(...)`.

Additionally, while the extension method *does* add argument validation, ensuring that e.g. only one API-1 is added, that means each `Add()` call is O(n) instead of (amortized) O(1).  This shouldn't matter with a list this small, but it is...odd.